src/CourseDashboard.cpp

```diff
@@ -82,3 +83,67 @@ void CourseDashboard::updateUser(User & user)
         break;
     }
 }
+
+std::string CourseDashboard::eMail()
+{
+        std::string email;
+        std::cout<<"Please enter a email."<<std::endl;
+        std::cin>>email;
+        std::cout << "e-mail: \"" << email<< "\"\n";
+
+    return email;
+}
+
+std::string CourseDashboard::passWord()
+{
+	std::string password1, password2;
+	std::cout << "Please enter your password." << std::endl;
+	std::cin >> password1;
+	std::cout << "Please re-enter your password." << std::endl;
+	std::cin >> password2;
+
+	while (password1 != password2)
+	{
+		std::cout << "Error! Passwords do not match." "\n";
+		std::cout << "Please enter your password." << std::endl;
+		std::cin >> password1;
+		std::cout << "Please re-enter your password." << std::endl;
+		std::cin >> password2;
+	}
+
+    return password1; 
+}
+
+void CourseDashboard::logIn()
+{
+    std::string emailKeyword = eMail();
+    std::string passwordKeyword = passWord();
+    std::string email, password;
+    std::string line;
+    std::fstream file;
+    file.open("plik.txt", std::ios::in);
+
+    if(!file.good())
+    {
+        std::cout<<"file doesn't exist";
+        exit(0);
+    }
+
+    while(!file.eof())
+    {
+        file>>email;
+        file>>password;
+
+        if(emailKeyword == email && passwordKeyword == password) 
+        {   
+            std::cout << "Login succeesful!" << std::endl; 
```

Comment:
how can I test if a login was successful or not? proper return value would be more useful.